core/controllers/feedback.py

```diff
@@ -244,3 +248,16 @@ def post(self):
         transaction_services.run_in_transaction(
             feedback_services.pop_feedback_message_references, user_id,
             len(references))
+
+
+class FeedbackMessageClear(base.BaseHandler):
```

Comment:
Usually we end all the handler names with "Handler". Maybe ClearFeedbackMessageHandler?